Formula/onlykey-agent.rb

```diff
@@ -0,0 +1,216 @@
+class OnlykeyAgent < Formula
+  include Language::Python::Virtualenv
+
+  desc "Middleware that lets you use OnlyKey as a hardware SSH/GPG device"
+  homepage "https://docs.crp.to/onlykey-agent.html"
+  url "https://files.pythonhosted.org/packages/02/c1/27c6cfbc5ee63fca91e37915d0182c0bfb988ca12362f01bcd5451e0ee10/onlykey-agent-1.1.14.tar.gz"
+  sha256 "096f20580ae112f57c1b9b279ed17728dc8e6f0fca301be586b9d976177f1523"
+  license "LGPL-3.0-only"
+
+  # https://docs.crp.to/onlykey-agent.html#installation
+  depends_on "rust" => :build # for cryptography
+  depends_on "gnupg"
+  depends_on "hidapi"
+  depends_on "libusb"
+  depends_on "openssl@1.1" # for cryptography
+  depends_on "python@3.10"
+  depends_on "six"
```

Comment:
You will want to add an entry in https://github.com/Homebrew/homebrew-core/blob/master/pypi_formula_mappings.json to avoid `six` being added by `brew [bump|bump-formula-pr]`.

Reply:
Thanks. Done in df549621a6869ba3fc89159a84f4caffbb2beb41